src/data.js

```diff
@@ -31,6 +31,12 @@ genotet.data.serverURL;
 genotet.data.uploadURL;
 
 /**
+ * User queries for user info and log.
+ * @type {string}
+ */
+genotet.data.userURL;
```

Comment:
I think I would combine user log and user authentication, so this would be the same as that branch?